src/cmd/troubleshoot/testutils.go

```diff
@@ -0,0 +1,23 @@
+package troubleshoot
```

Comment:
does GO know that this is only for testing? (so that its not built into the actual image)

I know that the `something_test.go` syntax does that, dunno about this

Reply:
checked, if these functions are only used in tests, then rename the file to `utils_test.go`

Reply:
As long as those methods are not used in production code, they won't be part of it. The linker should take care of this, just checked it (`strings -6 operator-binary  | grep getNullLogger`).
`utils_test.go` IMHO is misleading as it suggests to contain "test for utils" not "utils for test".

Reply:
could be in `logger_test.go` as it has to do with the logger (and then we dont use the forbidden `utils` word 😛 )